zcash_client_backend/src/tor.rs

```diff
@@ -24,20 +24,33 @@ impl Client {
     /// Preserving the contents of this directory will speed up subsequent calls to
     /// `Client::create`.
     ///
+    /// If `with_permissions` is `None`, the default from [`arti_client`] will be used
+    /// (enable permissions checks unless the `ARTI_FS_DISABLE_PERMISSION_CHECKS` env
+    /// variable is set).
+    ///
```

Comment:
Perhaps include an example of using `with_permissions` other than `None`.

Reply:
The type's methods have decent documentation, which is accessible through `rust-analyzer` in IDEs, so I think this should be fine as-is.